developer-docs-site/docs/guides/system-integrators-guide.md

```diff
@@ -62,16 +62,16 @@ Either of these methods will expose a [REST API service](./aptos-apis.md) at `ht
       <li>REST API: <a href="https://fullnode.testnet.aptoslabs.com">https://fullnode.testnet.aptoslabs.com</a></li>
       <li><a href="https://fullnode.testnet.aptoslabs.com/v1/spec#/">REST API Spec</a></li>
       <li>Indexer API: <a href="https://indexer-testnet.staging.gcp.aptosdev.com/v1/graphql">https://indexer-testnet.staging.gcp.aptosdev.com/v1/graphql</a></li>
-      <li><a href="https://cloud.hasura.io/public/graphiql?endpoint=https://indexer-testnet.staging.gcp.aptosdev.com/v1/graphql">Indexer Graphiql</a></li>
-      <li><a href="https://aptoslabs.com/testnet-faucet">Faucet Site</a></li>
+      <li><a href="https://cloud.hasura.io/public/graphiql?endpoint=https://indexer-testnet.staging.gcp.aptosdev.com/v1/graphql">Indexer GraphQL</a></li>
+      <li>Faucet API: <a href="https://faucet.devnet.aptoslabs.com">https://faucet.testnet.aptoslabs.com</a></li>
```

Comment:
The URL says testnet but you're actually pointing to devnet.

Reply:
I went ahead and fixed the link.